spec/models/member_spec.rb

```diff
@@ -1,8 +1,37 @@
 require 'spec_helper'
 
 describe Member do
+  subject { Fabricate(:member) }
+
+  it { should belong_to(:user) }
+  it { should belong_to(:organization) }
+  it { should have_one(:account) }
+  it { should delegate_method(:balance).to(:account).with_prefix }
+  it { should delegate_method(:gender).to(:user).with_prefix }
+  it { should delegate_method(:date_of_birth).to(:user).with_prefix }
+
+  it { should validate_presence_of(:organization_id) }
+  it { should validate_presence_of(:member_uid) }
+
+  pending "#offers" do
+    let(:member) { Fabricate(:member) }
+    let(:offer) { Fabricate(:offer, user: member.user) }
```

Comment:
I had tried that, already. The problem is that `member.organitzation` returns the name of the organization, tot the object. Anyway, I'll dig a little more and see what I can do.